sdk/src/main/scala/com/horizen/account/forger/AccountForgeMessageBuilder.scala

```diff
@@ -214,7 +250,7 @@ class AccountForgeMessageBuilder(mainchainSynchronizer: MainchainSynchronizer,
       new Array[Byte](MerkleTree.ROOT_HASH_LENGTH),
       new Array[Byte](MerkleTree.ROOT_HASH_LENGTH), // stateRoot TODO add constant
       new AddressProposition(new Array[Byte](Account.ADDRESS_SIZE)), // forgerAddress: PublicKeySecp256k1Proposition TODO add constant,
-      Long.MaxValue,
+      Account.INITIAL_BASE_FEE,
```

Comment:
need to pass `calculateBaseFee(nodeView, parentId)` to guarantee that block size is correct (at least not smaller). Or we can reserve again `BigInteger(Long.MaxValue).getBytes().size()`

Reply:
BigInteger(Long.MaxValue) would not be the max value, I changed it to a BigInteger with 256-bit